client/my-sites/plans/current-plan/jetpack-checklist/index.tsx

```diff
@@ -172,47 +134,166 @@ class JetpackChecklist extends PureComponent< Props > {
+								'::',
+								'/'
+							) }/wp-admin/admin.php?page=akismet-key-config` }
 							inProgress={ ! akismetFinished }
 							onClick={ this.handleTaskStart( { taskId: 'jetpack_spam_filtering' } ) }
 							target="_blank"
 						/>
 					) }
 
-					{ this.renderTaskSet( JETPACK_SECURITY_CHECKLIST_TASKS ) }
-					{ isEnabled( 'jetpack/checklist/performance' ) &&
-						this.renderTaskSet( JETPACK_PERFORMANCE_CHECKLIST_TASKS ) }
+					<Task
+						id="jetpack_monitor"
+						completed={ this.isComplete( 'jetpack_monitor' ) }
+						completedButtonText={ translate( 'Change', { context: 'verb' } ) }
+						completedTitle={ translate( 'You turned on Downtime Monitoring.' ) }
+						description={ translate(
+							"Monitor your site's uptime and alert you the moment downtime is detected with instant notifications."
+						) }
+						duration={ this.getDuration( 3 ) }
+						href={ `/settings/security/${ siteSlug }` }
+						onClick={ this.handleTaskStart( {
+							taskId: 'jetpack_monitor',
+							tourId: 'jetpackMonitoring',
+						} ) }
+						title={ translate( 'Downtime Monitoring' ) }
+					/>
+
+					<Task
+						id="jetpack_plugin_updates"
+						completed={ this.isComplete( 'jetpack_plugin_updates' ) }
+						completedButtonText={ translate( 'Change', { context: 'verb' } ) }
+						completedTitle={ translate( 'You turned on automatic plugin updates.' ) }
+						description={ translate(
+							'Choose which WordPress plugins you want to keep automatically updated.'
+						) }
+						duration={ this.getDuration( 3 ) }
+						href={ `/plugins/manage/${ siteSlug }` }
+						onClick={ this.handleTaskStart( {
+							taskId: 'jetpack_plugin_updates',
+							tourId: 'jetpackPluginUpdates',
+						} ) }
+						title={ translate( 'Automatic Plugin Updates' ) }
+					/>
+
+					<Task
+						id="jetpack_sign_in"
+						completed={ this.isComplete( 'jetpack_sign_in' ) }
+						completedButtonText={ translate( 'Change', { context: 'verb' } ) }
+						completedTitle={ translate( 'You completed your sign in preferences.' ) }
+						description={ translate(
+							'Manage your log in preferences and two-factor authentication settings.'
+						) }
+						duration={ this.getDuration( 3 ) }
+						href={ `/settings/security/${ siteSlug }` }
+						onClick={ this.handleTaskStart( {
+							taskId: 'jetpack_sign_in',
+							tourId: 'jetpackSignIn',
+						} ) }
+						title={ translate( 'WordPress.com sign in' ) }
+					/>
+
+					{ isEnabled( 'jetpack/checklist/performance' ) && (
+						<Task
+							id="jetpack_site_accelerator"
+							completed={ this.isComplete( 'jetpack_site_accelerator' ) }
+							completedButtonText={ translate( 'Configure' ) }
+							completedTitle={ translate(
+								'Site accelerator is serving your images and static files through our global CDN.'
+							) }
+							description={ translate(
+								'Serve your images and static files through our global CDN and whatch your page load time drop.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Serve your images and static files through our global CDN and watch your page load time drop.' )` *ES Score: 13*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/remove/jetpack-checklist-constants)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
This included a typo fixed by 2588fa7